decidim-admin/app/controllers/decidim/admin/participatory_processes_controller.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ParticipatoryProcessesController < ApplicationController
+      def index
+        @participatory_processes = collection
+        authorize @participatory_processes
```

Comment:
Code is not officially released in any version (it's only in the `master` branch` of the project), so we'll have to wait for the release. I've already asked them to make a new release: https://github.com/elabs/pundit/issues/423